schema/project.js

```diff
@@ -7,6 +7,12 @@ const statusQuery = status => query => Array.isArray(status)
   ? query.whereIn('projects.status', status)
   : query.where('projects.status', status);
 
+const getOwnAndCollaborations = licenceHolderId => query => query
+  .where({ licenceHolderId })
+  .orWhereExists(
+    Project.relatedQuery('profiles').where('profiles.id', licenceHolderId)
```

Comment:
```suggestion
    Project.relatedQuery('collaborators').where('profiles.id', licenceHolderId)
```

Reply:
To correspond with suggestion below.

Reply:
Also `licenceHolderId` is a terrible name for the argument here because it's finding the ones where the profile is _not_ the licence holder.

Reply:
good point :D